app/boards/shields/lotus58/Kconfig.defconfig

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) 2022 Temur Beissov
```

Comment:
Could we use `The ZMK Contributors` here as the copyright holder throughout? We prefer it to avoid churn when editing board/shield sources. This is your decision ultimately though!

Reply:
Yeah, for sure! I'll change it with the next commit.